File: SRC - PERL/00 - Estruturas Sequenciais/03.pl
#!/usr/bin/env perl
use v5.10;

use strict;
use warnings;

use utf8;
use open qw(:std :utf8);

# Faça um programa que receba o salário de um funcionário, calcule e mostre o novo salário, sabendo-se que este sofreu um aumento de 25%

print "Digite o salário > ";
my $salario = <>;

my $novo_salario = $salario + ($salario * 25 / 100);

printf "O novo salário é R\$ %.2f", $novo_salario;
print "\n";

File: startingPoint/02-EstruturaSequencial/Exercicios_Propostos/03.py
#!/usr/bin/python
# coding: latin-1

# 03 - Faça um programa que receba o salário de um funcionário, calcule e mostre o novo salário, sabendo-se que este sofreu um aumento de 25%

salario = int(input("Digite o valor do seu salário aqui: "))

aumento = (25 *(salario / 100))
novo_salario = (salario + aumento)

print("Seu novo salário será: ", novo_salario)
